cmd/server.go

```diff
@@ -103,6 +103,7 @@ to quickly create a Cobra application.`,
 
 		// Create a deadline to wait for.
 		var wait time.Duration
+		wait = time.Second * 15
```

Comment:
```suggestion
		wait := time.Second * 15
```

Reply:
should it be perhaps lower, since the default docker kill timeout is 10 seconds?

Reply:
Good point, have reduced to 10 seconds and applied your suggestion for defining directly.

On Kubernetes the default "terminationGracePeriodSeconds" is 30 seconds but this is for the whole pod not just the php-fpm_exporter sidecar, so 10 seconds here allows 20 seconds for php-fpm :+1: